alfresco-connector-test/src_test/alfresco/ecm/connector/test/TestAlfrescoECM.java

```diff
@@ -2,33 +2,37 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+
 import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
 import ch.ivyteam.ivy.bpm.engine.client.element.BpmProcess;
 import ch.ivyteam.ivy.bpm.engine.client.sub.SubProcessCallResult;
 import ch.ivyteam.ivy.bpm.exec.client.IvyProcessTest;
+import ch.ivyteam.ivy.environment.AppFixture;
 import ch.ivyteam.ivy.scripting.objects.List;
 import ch.ivyteam.ivy.security.ISession;
 
 /**
  * Rest Service functionality is mocked out here: {@link AlfrescoMock}
  */
 @IvyProcessTest
-public class TestAlfrescoECM
-{
-  private static final BpmProcess CALL_READ_DOCUMENTS =  BpmProcess.name("readDocuments");
- 
+public class TestAlfrescoECM {
+
+  private static final BpmProcess CALL_READ_DOCUMENTS = BpmProcess.name("readDocuments");
+
+  @BeforeEach
+  void setup(AppFixture fixture) {
+    fixture.config("RestClients.Alfresco.Url", AlfrescoMock.URI);
+  }
+
   @Test
   @SuppressWarnings("unchecked")
-  public void callReadDocuments(BpmClient bpmClient, ISession session)
-  {
-	  session.setActiveEnvironment("dev_axonivy");
```

Comment:
I assume that 80% of the new dev/nightly builds fail due to the absence of environments :)
... so I'll try to fix them all in a similar way.
![devBuildsForMarket](https://user-images.githubusercontent.com/15085693/235172306-861480e7-0e79-48a2-9bd6-102e0c2bba93.png)

Reply:
I plan to publish new releases, for all these repos, that were not compatible with 10+11.1 at once.
so It will get easier to use them with the current LE

Reply:
> I assume that 80% of the new dev/nightly builds fail due to the absence of environments :) ... so I'll try to fix them all in a similar way. ![devBuildsForMarket](https://user-images.githubusercontent.com/15085693/235172306-861480e7-0e79-48a2-9bd6-102e0c2bba93.png)

if my calculation is correct 15*100/36=41.6%
or the image is not up2date?